src/components/Auth/Login/Login.react.js

```diff
@@ -54,6 +54,7 @@ class Login extends Component {
       success: false,
       validForm: false,
       emailError: true,
+      passwordError: true,
```

Comment:
This whole logic can be implemented without using emailError, passwordError and validForm state variables. That would make this whole component more concise and elegant. Please make it happen.

Reply:
I don't think it can be implemented without using these state variables. Such as `passworfError` is used at line 172,181 and at many other lines.

Reply:
@samagragupta Sorry, I meant **validForm** data only. The only place it needs to be calculated is during rendering. So we won't need that as a state variable.

Reply:
It is being used at line 187 and 189.

Reply:
@samagragupta I take back what I said, the whole thing can be implemented without using emailError, passwordError and validForm. We have a very fresh implementation in chat.susi.ai. Go check it out and remove these redundant state variables.

Reply:
Also, make emailErrorMessage and passwordErrorMessage a part of the state.